dev/com.ibm.ws.jsonb_fat/test-applications/jsonbCDIapp/src/jsonb/cdi/web/JsonbCDITestServlet.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package jsonb.cdi.web;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import javax.inject.Inject;
+import javax.json.bind.Jsonb;
+import javax.servlet.annotation.WebServlet;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+
+@SuppressWarnings("serial")
+@WebServlet("/JsonbCDITestServlet")
+public class JsonbCDITestServlet extends FATServlet {
+
+    @Inject
+    CDIBean bean;
+
+    @Inject
+    Jsonb jsonb;
+
+    @Inject
+    Jsonb jsonb2;
+
+    @Test
+    public void testReuseInjectedJsonb() {
+        assertNotNull(jsonb);
+        assertNotNull(jsonb2);
+        assertNotNull(bean.getJsonb());
+        assertEquals("Two Jsonb injection points should inject the same object instance", jsonb.toString(), jsonb2.toString());
```

Comment:
If the same object instance, why the need to convert to String before comparing? (applies to next line as well)

Reply:
agreed, changed to comparison of raw object instances